pkgs/tools/games/ajour/default.nix

```
@@ -0,0 +1,68 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, autoPatchelfHook
+, cmake
+, pkg-config
+, expat
+, openssl
+, libxkbcommon
+, vulkan-headers
+, vulkan-loader
+, wayland
+, x11
+, xorg
+, python3
+}:
+
+let
+  rpathLibs = [
+    libxkbcommon
+    vulkan-loader
+    wayland
+  ];
```

I noticed that X11 wasn't listed as a load time dependency through ldd, so I switched over to sway and found that these also need to be added to `rpathLibs` for X11 support:
```suggestion
  rpathLibs = [
    libXcursor
    libXi
    libxkbcommon
    libXrandr
    vulkan-loader
    wayland
    x11
  ];
```

---

I indeed only tested this on wayland at this point. Thanks for coming up with the list for X11.